Dockerfile

```diff
@@ -1,21 +1,23 @@
-# Get the base Ubuntu image from Docker Hub
+# Get the base test environment image from Docker Hub
 FROM oswasabi/cadg-test-env:latest
 
-# Set the folder name to be the same as the lesson name
-ARG DIR=cadg
+# Set the working directory
+WORKDIR "/usr/src/cadg"
 
 # Copy the current folder which contains C++ source code to the Docker image under /usr/src
-COPY . "/usr/src/${DIR}"
-
-# Specify the working DIRectory
-WORKDIR "/usr/src/${DIR}"
+COPY . .
```

Comment:
why "COPY . ."? doesn't this just copy everything here and paste it here?

Reply:
I answered this on the other pull request as I saw that one first. This simply copies the code into the docker container.

Reply:
Thanks, Kris. This PR has my approval.